src/AirshipBindings.iOS/ApiDefinition.cs

```diff
@@ -1396,6 +1396,7 @@ interface UADisposable
 
 		// -(void)dispose;
 		[Export ("dispose")]
+        [New]
```

Comment:
What is this annotation for?

Reply:
So Xamarin adds a `Dispose()` to its version of NSObject. This says our `dispose()` is a new method, not an override.

Reply:
ah interesting. I'm guessing this is just one of those things we need to special case, like "@object"

Reply:
Yes. Or maybe we can add a notation in the code that is picked up by Bluntie?

Reply:
Perhaps. I think a reasonable pattern to follow is to special case things that we know ahead of time to conflict with c#, such as reserved keywords, and maybe introduce hints on the obj-c side for stuff that sourcekitten has difficulty parsing. But all suggestions are welcome.

Reply:
I think we are likely to get some stuff right and some stuff wrong at first. And some stuff will become right or wrong over time as our SDK and Xamarin change. But I'm so looking forward to being able to specify these things in some other way than manually editing `ApiDefinitions.cs` every time!!